OpenRA.Mods.Common/Util.cs

```diff
@@ -77,16 +77,21 @@ public static WPos BetweenCells(World w, CPos from, CPos to)
 			return WPos.Lerp(w.Map.CenterOfCell(from), w.Map.CenterOfCell(to), 1, 2);
 		}
 
-		/* pretty crap */
 		public static IEnumerable<T> Shuffle<T>(this IEnumerable<T> ts, MersenneTwister random)
 		{
-			var items = ts.ToList();
-			while (items.Count > 0)
+			// Fisher�Yates
```

Comment:
@MonoDevelop does not like the special char as it seems:

![image](https://cloud.githubusercontent.com/assets/756669/13032941/02994068-d306-11e5-97ac-e6dd6af290e2.png)